examples/userapi/07_sparse_operations.ipynb

```diff
@@ -0,0 +1,965 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    ax.imshow(u.data[1], vmin=0, vmax=2, cmap="jet", extent=[0,1,0,1])
</pre><p>A colourbar is needed here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/2252/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>